src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/HearingOrderController.java

```diff
@@ -57,4 +60,54 @@
 
         return ResponseEntity.ok(AboutToStartOrSubmitCallbackResponse.builder().data(caseData).build());
     }
+
+    @PostMapping(path = "/hearing-order/approval-start", consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "Starts hearing order approval")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Callback was processed successfully or in case of an error message is attached to the case",
+            response = AboutToStartOrSubmitCallbackResponse.class),
+        @ApiResponse(code = 400, message = "Bad Request"),
+        @ApiResponse(code = 500, message = "Internal Server Error")})
+    public ResponseEntity<AboutToStartOrSubmitCallbackResponse> startHearingOrderApproval(
+        @NotNull @RequestBody @ApiParam("CaseData") CallbackRequest callback) {
+        CaseDetails caseDetails = callback.getCaseDetails();
+        validateCaseData(callback);
+
+        Map<String, Object> caseData = caseDetails.getData();
+        Optional<DraftDirectionOrder> draftDirectionOrderCollectionTail = hearingOrderService.draftDirectionOrderCollectionTail(caseDetails);
+        if (draftDirectionOrderCollectionTail.isPresent()) {
```

Comment:
personally I would prefer this logic to be in the service method. that way the controller unit tests only needs to verify the service was called and the service unit tests can validate the contents of the case data